src/test/java/com/mongodb/hibernate/cfg/spi/MongoConfigurationContributorTests.java

```diff
@@ -59,6 +63,28 @@ void exceptionPropagates() {
         assertSame(expected, actual.getCause());
     }
 
+    @Test
+    @DisplayName(
+            "MongoConfigurationContributor is called once per StandardServiceRegistry, different MongoConfigurator instances passed")
+    @SuppressWarnings("try")
+    void invocationsAndMongoConfiguratorInstances() {
```

Comment:
This test is former `StandardServiceRegistryScopedStateTests.mongoConfigurationContributorInvocationsAndMongoConfiguratorInstances`. I moved it, and simplified: we don't need to close `bootstrapServiceRegistry`, `standardServiceRegistry1`, `standardServiceRegistry2`, because we have session factories successfully created here, and we close them, which causes the service registries to be automatically-closed (see https://docs.hibernate.org/orm/6.6/javadocs/org/hibernate/boot/registry/BootstrapServiceRegistryBuilder.html#disableAutoClose() and https://docs.hibernate.org/orm/6.6/javadocs/org/hibernate/boot/registry/StandardServiceRegistryBuilder.html#disableAutoClose()).